google/cloud/forseti/common/gcp_api/_base_repository.py

```diff
@@ -488,6 +488,25 @@ def execute_query(self, verb, verb_arguments):
     def _execute(self, request):
         """Run execute with retries and rate limiting.
 
+        Args:
+            request (object): The HttpRequest object to execute.
+
+        Returns:
+            dict: The response from the API.
+        """
+        try:
+            return self._execute_request(request)
+        except exceptions.RefreshError as e:
+            # If there is problem refreshing the token, we will recreate a new
+            # Credential object and use that to refresh the request.
+            LOGGER.exception(e)
+            self._credentials, _ = api_helpers.get_google_default_credentials()
+            self._credentials.refresh(request)
```

Comment:
It seems like we need to set the `CLOUD_SCOPES` here.  Maybe centralize it in the `api_helpers.get_google_default_credentials()` everywhere?

Reply:
Updated